src/com/amazon/ion/impl/_Private_IonBinaryWriterBuilder.java

```diff
@@ -223,6 +223,13 @@ public _Private_IonBinaryWriterBuilder withLocalSymbolTableAppendDisabled()
         return b;
     }
 
+    @Override
+    public _Private_IonBinaryWriterBuilder withDefaultBlockSize(int size) {
```

Comment:
Remove the word "default" from the name of these methods. The user is setting the block size; the default block size is 32K.